codemagic.yaml

```diff
@@ -10,13 +10,6 @@ definitions:
       cache_paths:
         - $HOME/.gradle/caches
   scripts:
-    - &set_up_google_services_files_from_environment_variables
-      name: Set up google-services.json on template-compose
-      script: |
-        mkdir -p template-compose/app/src/production
-        echo $GOOGLE_SERVICES_JSON > template-compose/app/src/production/google-services.json
-        mkdir -p template-compose/app/src/staging
-        echo $GOOGLE_SERVICES_JSON_STAGING > template-compose/app/src/staging/google-services.json
```

Comment:
Should we also clean up all unnecessary steps in https://github.com/search?q=repo%3Animblehq%2Fandroid-templates+GOOGLE_SERVICES_JSON&type=code